MapboxNavigation/DistanceFormatter.swift

```diff
@@ -7,32 +6,32 @@ let yardsPerMile = 1_760.0
 let feetPerMile = yardsPerMile * 3.0
 
 /// Provides appropriately formatted, localized descriptions of linear distances.
-public class DistanceFormatter: LengthFormatter {
+class DistanceFormatter: LengthFormatter {
```

Comment:
Why are there separate DistanceFormatter classes in MapboxCoreNavigation and MapboxNavigation? Let’s delete this one and have MapboxNavigation use the one out of MapboxCoreNavigation.